### DISCUSSION TITLE
Missing/Error on ticker AMKBY

### DISCUSSION THREAD

**Discussion by ori-maci** [OpenBB-finance/OpenBB#5038] [General]
Hello,

When I run the following in openBB terminal, I get this error for ticker AMKBY:
![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/10456511/daa5d50c-0b61-4ca6-9316-17d0a9fa3bf5)

For these other commands on the SDK, I see errors like: 

```python
openbb.stocks.fa.data(symbol=AMKBY) # There was an error connecting to the API. Please try again later.
overview_df_data = openbb.stocks.fa.overview(symbol=AMKBY) # No data found from Alpha Vantage
alpha_balance_df_data = openbb.stocks.fa.balance(
            symbol=AMKBY, source="AlphaVantage"
        ) # No data found from Alpha Vantage, looking in Yahoo Finance
```

Can someone confirm if this is a valid bug? Or is it a missing data issue? If so, will this data be populated into OpenBB?

**Comment by IgorWounds** [comment]
Hi,

Thanks for raising this discussion! 

This is an API-related error as some of the sources don't support the AMKBY ticker. Please try changing the default source which should result in it working. Attached is a screenshot with the commands working on my end:

`/stocks/fa/ $ overview --source YahooFinance`

![Screenshot 2023-05-23 at 17 51 50](https://github.com/OpenBB-finance/OpenBBTerminal/assets/74266147/504a4093-6706-4021-8a0b-61dc29d9efca)

`/stocks/fa/ $ balance --source YahooFinance`

![Screenshot 2023-05-23 at 17 59 45](https://github.com/OpenBB-finance/OpenBBTerminal/assets/74266147/c1c54154-bacf-42e2-afd2-8afbfc12bd19)
